python/cugraph/cugraph/community/ecg.py

```diff
@@ -119,6 +108,19 @@ def ecg(
 
     input_graph, isNx = ensure_cugraph_obj_for_nx(input_graph)
 
+    if isNx:
+        warning_msg = (
+            " We are deprecating support for handling "
+            "NetworkX types in the next release."
+        )
+        warnings.warn(warning_msg, UserWarning)
```

Comment:
This message doesn't hurt but isn't needed since `ensure_cugraph_obj_for_nx()` will handle the deprecation warning in this PR: https://github.com/rapidsai/cugraph/pull/4493/files#diff-62286c02304119d912dbd0491893a58c1dea11bd0addcc80fadb3d129d21029f